modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm

```diff
@@ -1725,3 +1725,12 @@ MAPPING_DIRECTIONAL_HELPERS(/obj/structure/sign/poster/contraband/korpstech, 32)
 	return ..()
 
 #undef NOBILITY_DRESSCOAT_WASHING_CREDITS_NEEDED
+
+/obj/item/clothing/head/anubite
+	name = "Anubite Headpiece"
+	desc = "A dark coloured headpiece with golden accents. It's features seem reminiscent of the god Anubis."
+	icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/head.dmi'
+	icon_state = "anubite_headpiece"
+	worn_icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/hats.dmi'
+	worn_icon_state = "anubite_headpiece"
```

Comment:
```suggestion
```
It shouldn't _need_ this second define, "icon_state" will by default apply to the worn_icon_state...
Hopefully. Some base items break it. If clothing/head does then... welp.

Reply:
I swear when I originally tested it, I got an empty sprite when wearing it. Seems to work fine now though so.